llama_hub/wikipedia/README.md

```diff
@@ -15,4 +17,7 @@ loader = WikipediaReader()
 documents = loader.load_data(pages=['Berlin', 'Rome', 'Tokyo', 'Canberra', 'Santiago'])
 ```
 
-This loader is designed to be used as a way to load data into [LlamaIndex](https://github.com/jerryjliu/gpt_index/tree/main/gpt_index) and/or subsequently used as a Tool in a [LangChain](https://github.com/hwchase17/langchain) Agent. See [here](https://github.com/emptycrown/llama-hub/tree/main) for examples.
+This loader is designed for loading data into [LlamaIndex](https://github.com/run-llama/llama_index/tree/main/llama_index)
+and/or subsequently used as a Tool in a [LangChain](https://github.com/langchain-ai/langchain) Agent.
```

Comment:
missing remove this line

Reply:
I don't quite understand, do you mean:

- Revert this change
- Remove lines 20 and 21 entirely from the `README`

Just lemme know which one you intended?

Reply:
> and/or subsequently used as a Tool in a [LangChain](https://github.com/langchain-ai/langchain) Agent.

remove only the line 21, since it can't be used as a tool, since it's only for loading data

Reply:
Alright, made the change.  Btw, feel free to directly commit to this branch if needed